orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/IgorService.java

```diff
@@ -85,4 +86,13 @@ GoogleCloudBuild runGoogleCloudBuildTrigger(
       @Path("account") String account,
       @Path("triggerId") String triggerId,
       @Body GoogleCloudBuildRepoSource repoSource);
+
+  @GET("/delivery-config/manifest")
+  Map<String, Object> getDeliveryConfigManifest(
+      @Query("scmType") String scmType,
+      @Query("project") String project,
+      @Query("repository") String repository,
```

Comment:
Not sure if we want to be consistent here, but there are some naming conventions that other services are using them already, like repoSlug instead of repository. You can view it here https://boost.test.netflix.net/swagger-ui.html#/builds-controller/findBuildsUsingGET

Reply:
That's a good question. I noticed that and, in fact, had originally named everything following that convention. Then I realized that "slug" is an [Atlassian term](https://confluence.atlassian.com/bitbucket/what-is-a-slug-224395839.html), apparently, whereas this API is supposed to be SCM-type agnostic. I don't have a super strong opinion, although at this point igor has been merged with the different names as well (except in the `StashClient`, which is Stash-specific).

Reply:
Hmm interesting. I vote to keep it consistent with boost, as we might use your code in the future if we'll want to add a ci stage for example. any thoughts @emjburns ?